calico/reference/cni-plugin/configuration.md

```diff
@@ -4,18 +4,9 @@ description: Details for configuring the Calico CNI plugins.
 canonical_url: '/reference/cni-plugin/configuration'
 ---
 
-{% tabs %}
-  <label:Operator,active:true>
-<%
-
-The {{site.prodname}} CNI plugins do not need to be configured directly when installed by the operator. For a complete operator 
-configuration reference, see [the installation API reference documentation][installation].
-
-%>
-
-  <label:Manifest>
-<%
-
+>**Note**: The {{site.prodname}} CNI plugins do not need to be configured directly when installed by the operator.
+>For a complete operator configuration reference, see [the installation API reference documentation][installation].
+{: .alert .alert-info}
```

Comment:
Should we consider using `.alert-danger` to make sure the users don't just scroll past this and inadvertently break things? Especially since operator is the default... @caseydavenport @mgleung what do you think?

Reply:
I think it would be better, but since we want to restructure soon anyway I'm not too bothered. Let's get this in and then I might just restructure.